### ISSUE TITLE
What about server side validation for if the username exists?

### ISSUE THREAD

**Issue by smorsi** [tugberkugurlu/MvcRemoteValidationSample#1]

**Comment by tugberkugurlu** [comment]
@smorsi Yes, you need to perform the same business logic at the server side in order to make sure this as user can disable/suppress the client side validation. It has been too long so my memory is kind of rusty on this but as far as I remember, it doesn't work by default with the validation attribute on the server side.
